staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_extractTag(t *testing.T) {
+	testCases := []struct {
+		comments []string
+		expect   []string
+	}{
```

Comment:
It's good to have unique names for each test case. I personally prefer `map[string]struct{...}` because then the compiler ensures that the names are unique. Avoid spaces in the name, otherwise the name in the code does not match what `go test` uses when using the name as subtest (see below).